typed_python/compiler/tests/python_object_of_type_compilation_test.py

```diff
@@ -362,6 +365,9 @@ def test_bool_cast_and_conv(self):
             (B4, B4.a(s='')),
             (B4, B4.a(s='a')),
             (B5, B5.a(s='')),
+            # TODO: The line below segfaults in the specialized entrypoint case,
+            # due to attempting set iteration on this type
+            # (PointerTo(Int64), x0.pointerUnsafe(0)),
```

Comment:
please file a ticket so we don't forget about it.

Reply:
Created.